src/main/resources/db/migration/V1_8__kontor.sql

```diff
@@ -0,0 +1,33 @@
+CREATE TABLE BEHANDLER_DIALOGMELDING_KONTOR
```

Comment:
Nå har vi fra før av BehandlerDialogmelding, DialogmeldingToBehandlerBestilling, DialogmeldingFromBehandler og BehandlerDialogmeldingArbeidstaker. Det er vel ikke alle disse som har dialogmeldinginfo i seg. Blir det lettere å skille disse domeneordene fra hverandre hvis vi tar bort dialogmelding der det ikke trengs? Kan denne hete BEHANDLER_KONTOR feks?

Reply:
Det går an, men i så fall synes jeg DIALOGMELDING bør fjernes fra navnene til de to andre tabellene også (dvs BEHANDLER_DIALOGMELDING og BEHANDLER_DIALOGMELDING_ARBEIDSTAKER).

Reply:
Enig, det tror jeg også blir fint

Reply:
Støtter dette

Reply:
OK, kan fikses i en egen PR.